01_easy_algorithms/08_recursion_length_of_string/08_recursion_length_of_string.rb

```diff
@@ -0,0 +1,4 @@
+def length(str)
+  return 0 if str == ""
+  return length(str[1..(str.length)]) + 1
```

Comment:
You don't need to specify return when it's the last line. Also to remove a char from the string, use more concise Ruby methods like str.chop.